src/renderer/atlas/AtlasEngine.cpp

```diff
@@ -634,16 +633,20 @@ void AtlasEngine::_flushBufferLine()
         const auto indicesCount = row.glyphIndices.size();
         if (indicesCount > initialIndicesCount)
         {
-            row.mappings.emplace_back(std::move(mappedFontFace), _p.s->font->fontSize * scale, gsl::narrow_cast<u32>(initialIndicesCount), gsl::narrow_cast<u32>(indicesCount));
+            row.mappings.emplace_back(std::move(mappedFontFace), gsl::narrow_cast<u32>(initialIndicesCount), gsl::narrow_cast<u32>(indicesCount));
         }
     }
 }
 
-void AtlasEngine::_mapCharacters(const wchar_t* text, const u32 textLength, u32* mappedLength, float* scale, IDWriteFontFace2** mappedFontFace) const
+void AtlasEngine::_mapCharacters(const wchar_t* text, const u32 textLength, u32* mappedLength, IDWriteFontFace2** mappedFontFace) const
 {
     TextAnalysisSource analysisSource{ text, textLength };
     const auto& textFormatAxis = _api.textFormatAxes[static_cast<size_t>(_api.attributes)];
 
+    // We don't read from scale anyways.

```

Comment:
then why keep it

Reply:
oh

Reply:
Yeah, the parameter isn't optional unfortunately.